label.py

```diff
@@ -29,6 +32,32 @@
     random.shuffle(images_in_all_labels)
     images_to_show = images_not_in_all_labels + images_in_all_labels
 
+if args.model:
```

Comment:
It would be nice to move the evaluation code into a module, so that it can be shared by label.py and by whatever module we will implement in the future as a backend for the UI.